### DISCUSSION TITLE
Free Whatsapp Gateway and SMS Gateway

### DISCUSSION THREAD

**Discussion by ibnux** [hotspotbilling/phpnuxbill#13] [Show and tell]
i create Gateway to be used using PHPNuxBill

[wa.nux.my.id](https://wa.nux.my.id/login)

Why its free?

It use your own phone and number.

**Comment by wakidanwar** [comment]
Thanks Mr. Ibnu. This is such powerfull apps.

**Comment by umblox** [comment]
sejak tadi malam wa gagal masuk, di wa.nux.my.id status connected, saya disconnect dr apk wa, berharap bisa scan ulang tapi tetep status nya connected. coba beli paid version tetep ga bisa masuk om🙏

**Reply by ibnux** [reply]
Sudah fix. Ada update dari Whatsapp dan aplikasi servernya

**Reply by alucard2303** [reply]
I put my number with the country code and I never get any code. I'm from Mexico. Please help me.

**Comment by skylinkscure** [comment]
bagaimana supaya nomer wa yg kita daftarkan terhindar dari baned whatsapp?
sy kemarin kebaned karena dianggap spam. dan pesan notifikasi tidak dikirim keseluruhan pelanggan yg harusnya dinotifikasi.
mohon bimbingan.

**Reply by ibnux** [reply]
Nomornya diikutkan ke grup grup WhatsApp yang aktif 


Cari aja grup info saham atau apa 😆

**Reply by skylinkscure** [reply]
Siap kak. Masih nubi soalnya. udah korban 1 nomer

**Comment by DinarArisandi** [comment]
aplikasinya tidak bisa digunakan

https://wa.nux.my.id

This site can’t be reached
wa.nux.my.id took too long to respond.
Try:

Checking the connection
[Checking the proxy and the firewall](chrome-error://chromewebdata/#buttons)
[Running Windows Network Diagnostics](javascript:diagnoseErrors())
ERR_CONNECTION_TIMED_OUT


The connection has timed out

The server at wa.nux.my.id is taking too long to respond.
What can you do about it?

    The site could be temporarily unavailable or too busy. Try again in a few moments.
    If you are unable to load any pages, check your computer’s network connection.
    If your computer or network is protected by a firewall or proxy, make sure that Firefox is permitted to access the web.
